gocd/templates/bash/check-github-runs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/devinfra/scripts/checks/githubactions/checkruns.py \
+  getsentry/chartcuterie \
+  "${GO_REVISION_CHARTCUTERIE_REPO}" \
+  build
```

Comment:
I wish this was a named argument instead of a positional argument. I know from inference that this is the name of the GHA workflow, but still

Reply:
i updated the named the arguments using [here](https://github.com/getsentry/devinfra-deployment-service).